conda_env/env.py

```diff
@@ -51,7 +51,9 @@ def from_environment(name, prefix, no_builds=False):
     if len(pip_pkgs) > 0:
         dependencies.append({'pip': ['=='.join(a.rsplit('-', 2)[:2]) for a in pip_pkgs]})
 
-    channels = config.get_rc_urls()
+    # conda uses ruamel_yaml which returns a ruamel_yaml.comments.CommentedSeq
+    # this doesn't dump correctly using pyyaml
```

Comment:
Well, I was *going* to make a patch that replaced pyyaml with ruamel_yaml, but conda-env apparently has a custom wrapper around pyyaml, so it's not a straight-forward replacement.  Will take a bit more time than just switching out the library.

If you or someone else wants to do that, please be my guest.

Today and tomorrow I'm focused on getting conda canary 4.2.0 out.